data/remnant/remnant logistics.txt

```diff
@@ -0,0 +1,295 @@
+			choice
+				`	"Won't your pilots be upset at the competition?"`
+					goto competition
+				`	"Thanks for the opportunity."`
+					goto thanks
+			label competition
+			`	Eon smiles at the question. "You should talk to the other pilots and see. But, I think the ancestral space approach is different. We work more like how I imagine the navy does in the Republic. The ships are owned by the Remnant, but each captain is responsible for the ship, the crew, and the transport work that needs to be done. I think they will be happy to see new homes being built for families, and, "he winks at you, "spending more time with their own."`
+				goto thanks
+			label thanks
+			`	Before you can ask more about transport jobs, Eon takes out his commlink and taps on it for a few minutes. "I've added you to the job board. You should be able to see our current job list for each system when you check."`
+			`	You chat with Eon about how you've outfitted the <ship> as you finish your...coffee. Then, his commlink chirps. As he stands, he signs what you think means 'Goodbye' as he sings that he has to leave for another meeting. He walks off quickly with his head down looking at the datapad. Your commlink notifies you that <payment> has been added to your account.`
+
+trade
+	commodity "remnant civil"
+		"power distribution controls"
+		"sewage treatment modules"
+		"road grading and surfacing equipment"
+		"modular crane components"
+		"high tensile cables"
+		"earth moving equipment"
+		"heavy machinery tools"
+		"high voltage electrical diagnostic equipment"
+		"environmental heating and cooling systems"
+		"high volume pumps"
+		"environmental monitoring and reporting modules"
+		"construction robots"
+		"construction matrix mixers and extruders"
+		"worker housing modules"
+		"drilling equipment"
+		"waste recycling equipment"
+		"civil defense emergency supplies"
+		"portable machine shop modules"
+		"local network repeaters and uplinks"
+
+mission "Remnant: Logistics Civil Engineering"
+	name "Civil Engineering Delivery"
+	description `Engineers on <planet> need <cargo> delivered. Logistics adjustment is <payment>.`
+	cargo "remnant civil" 30 80
+	job
+	repeat
+	to offer
+			not "Remnant: Ssil Vida Active"
+			not "flagship planet: Ssil Vida"
+			has "Remnant: Logistics 1: done"
+			random < 51
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+		not distance 0
+		not planet "Ssil Vida"
+		not planet "Esquiline"
+	on visit 
+		dialog phrase "generic cargo on visit"
+	on complete
+		dialog `A project engineer inspects the <cargo> and signs off on your payment of <payment>.`
+		payment 50000 10
```

Comment:
```suggestion
		payment 50000 10
```
Same as above with this payment being a bit too high. I'd suggest the first number be somewhere in the range of 40,000 and increase the second number by 5-10ish, but I'm willing to workshop if you have other ideas. Alternatively, something like this:
```suggestion
		payment
		payment -5000
```
This would just subtract 5000 credits from whatever the normal payment for a job would be. Up to you.